README.md

```diff
@@ -1,36 +1,31 @@
-[![Manager](https://github.com/thoughtworks/maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/manager.yml)
-[![Gateway](https://github.com/thoughtworks/maeve-csms/workflows/Gateway/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/gateway.yml)
+[![Manager](https://github.com/thoughtworks/Maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/Maeve-csms/actions/workflows/manager.yml)
+[![Gateway](https://github.com/thoughtworks/Maeve-csms/workflows/Gateway/badge.svg)](https://github.com/thoughtworks/Maeve-csms/actions/workflows/gateway.yml)
 
 # MaEVe
 
-MaEVe is an EV charge station management system (CSMS). It began life as a simple proof of concept for
-implementing ISO-15118-2 Plug and Charge (PnC) functionality and remains a work in progress. It is hoped that over
-time it will become more complete, but already provides a useful basis for experimentation.
+Maeve is an EV Charge Station Management System (CSMS). It began life as a simple proof of concept for
+implementing ISO-15118-2 Plug and Charge (PnC) functionality and remains a work in progress. Over
+time it will become more complete. It provides a useful basis for experimentation with [Everest](https://github.com/EVerest).
 
-The system currently integrates with [Hubject](https://hubject.stoplight.io/) for PnC functionality.
-
-## Table of Contents
-- [Documentation](#documentation)
-- [Getting Started](#getting-started)
-- [Configuration](#configuration)
-- [Contributing](#contributing)
-- [License](#license)
+Maeve integrates with [Hubject](https://hubject.stoplight.io/) for Plug and Charge (PnC) functionality.
 
 ## Documentation
-MaEVe is implemented in Go 1.20. Learn more about MaEVe and its existing components through this [High-level design document](./docs/design.md).
+
+Learn more about Maeve and its existing components through this [High-level design document](./docs/design.md).
 
 ## Pre-requisites
 
-MaEVe runs in a set of Docker containers. This means you need to have `docker`, `docker-compose` and a docker daemon (e.g. docker desktop, `colima` or `rancher`) installed and running.
+Maeve runs in a set of Docker containers. This means you need to have `docker`, `docker-compose` and a docker daemon (e.g. [colima](https://github.com/abiosoft/colima)) installed and running.
 
 ## Getting started
 
 To get the system up and running:
 
 1. `(cd config/certificates && make)`
-2. Run the [./scripts/run.sh](./scripts/run.sh) script with the same token to run all the required components - again, don't forget the quotes around the token
+2. `./scripts/run.sh`
+
+Charge stations connect to Maeve using Web sockets:
 
-Charge stations can connect to the CSMS using:
```

Comment:
Where else is this information provided?

Reply:
"Charge stations connect to Maeve using Web sockets:"